.github/renovate.json5

```diff
@@ -162,4 +162,4 @@
     },
   ],
   platformAutomerge: true,
-}
+}
```

Comment:
@luhi-DT  is it something we manually touch recently?

When I run locally script:

<img width="736" alt="image" src="https://github.com/user-attachments/assets/179fc202-2b58-4a7a-ad16-ee4fc8f3754c">